uast/uast.go

```diff
@@ -30,6 +30,28 @@ const (
 	Expression
 	Statement
 
+	// Unary operators
+	// ++x
+	UnaryPreIncrement
+	// x++
+	UnaryPostIncrement
+	// --x
+	UnaryPreSubstract
```

Comment:
`Substract` could be replaced with `Decrement`, to not confuse it with binary substraction operation.